### PR TITLE
Issue #11985: Add EJB 4.0-RC1 feature and project

### LINKED ISSUE #11985: Add feature for Jakarta Enterprise Beans API 4.0.0-RC1
Create the projects for building the bundle and feature for Jakarta Enterprise Beans API 4.0.0-RC1

### PR DESCRIPTION
**Pull request by tkburroughs**
Add projects and features required to include the EJB 4.0-RC1 API
JAR in the builds. At least one public feature is required, so
added ejbLite-4.0, which won't work just yet, but will allow
other components to include the EJB API dependency as needed.

fixes #11985

### CODE REVIEW

**Comment by tkburroughs** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/ejbLite-4.0/resources/l10n/com.ibm.websphere.appserver.ejbLite-4.0.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for Jakarta Enterprise Beans written to the Enterprise Beans Lite subset of the Jakarta Enterprise Beans 4.0 specification.
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/ejbLite-4.0/resources/l10n/com.ibm.websphere.appserver.ejbLite-4.0.properties:17]: suggested edit:
This feature enables support for Jakarta Enterprise Beans that are written to the Enterprise Beans Lite subset of the Jakarta Enterprise Beans 4.0 specification.

**Comment by tkburroughs** [comment]
#libbybot

File: dev/com.ibm.websphere.appserver.features/visibility/public/ejbLite-4.0/resources/l10n/com.ibm.websphere.appserver.ejbLite-4.0.properties (hunk 1 above)
Comment (tkburroughs) [reply dev/com.ibm.websphere.appserver.features/visibility/public/ejbLite-4.0/resources/l10n/com.ibm.websphere.appserver.ejbLite-4.0.properties]: @dmuelle Updated as suggested

**Comment by tkburroughs** [comment]
#run-libbybot

**Comment by tkburroughs** [comment]
#build

**Comment by tkburroughs** [comment]
#build

**Comment by tkburroughs** [comment]
#build

**Comment by tkburroughs** [comment]
Build had 2 known intermittent FAT failures.... merging.

**Pull request merged**